sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/LogsHelper.cs

```diff
@@ -66,16 +66,6 @@ internal static List<TelemetryItem> OtelToAzureMonitorLogs(Batch<LogRecord> batc
         {
             string? message = logRecord.FormattedMessage;
 
-            // Both logRecord.State and logRecord.StateValues will not be set at the same time for LogRecord.
```

Comment:
Will there be an impact if customer sets the `ParseStateValues` to false?

Reply:
as long as we dont crash we are okay. I think we'd still send TraceTelemetry, with no custom props on it.

Reply:
Yes, we will not crash (only the custom dimensions will be missing)